recipes/autoconf/all/conanfile.py

```diff
@@ -9,30 +11,31 @@ class AutoconfConan(ConanFile):
     description = "Autoconf is an extensible package of M4 macros that produce shell scripts to automatically configure software source code packages"
     topics = ("conan", "autoconf", "configure", "build")
     license = ("GPL-2.0-or-later", "GPL-3.0-or-later")
-    exports_sources = "patches/**"
     settings = "os", "arch", "compiler"
+
+    exports_sources = "patches/*"
     _autotools = None
 
     @property
     def _source_subfolder(self):
         return os.path.join(self.source_folder, "source_subfolder")
 
```

Comment:
@property
    def _settings_build(self):
        return self.settings_build if hasattr(self, "settings_build") else self.settings

Reply:
Do we want to go all the way and do the following as well?
```diff
--        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
++        self._autotools = AutoToolsBuildEnvironment(self, win_bash=self._settings_build == "Windows")
```

Reply:
According to @jgsogo (can't remember where is the message), it doesn't matter in `build()`.

Reply:
It probably doesn't matter for the central c3i server that forwards package builds to linux/macos/windows builders.

Pros of using `self._settings_build` in `build` is that there is one truth for the builds os (=settings_build).
Contra is that it's easier to write `tools.os_info.is_windows` the implementation of `_settings_build` + `self._settings_build.os == "Windows"`

Reply:
This line isn't modified anymore by this pr.